clients.yml

```diff
@@ -410,7 +410,7 @@ project/releng/generic-worker/datacenter-gecko-t-osx-1015-r8:
 project/releng/generic-worker/datacenter-mozilla-b-1-osx:
   description: Datacenter MacOS dedicated developer and automation worker.
   scopes:
-    - assume:worker-type:releng-hardware/gecko-t-osx*
+    - assume:worker-type:releng-hardware/mozilla-b*
```

Comment:
Sorry, should've mentioned this in the original review...this should be more tightly scoped too. As it stands now, this could assume `mozilla-b-3-osx` as well as `mozilla-b-1-osx`.